config.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
 
-  kmsEncryptedHookUrl: process.env.KMS_ENCRYPTED_HOOK_URL, // encrypted slack webhook url
-  unencryptedHookUrl: process.env.UNENCRYPTED_HOOK_URL,    // unencrypted slack webhook url
-  slackChannel: process.env.SLACK_CHANNEL,                 // slack channel to send a message to
-  slackUsername: process.env.SLACK_USERNAME,               // "AWS SNS via Lamda", // slack username to user for messages
-  icon_emoji: process.env.ICON_EMOJI,                      // slack emoji icon to use for messages
-  orgIcon: process.env.ORG_ICON,                           // url to icon for your organization for display in the footer of messages
-  orgName: process.env.ORG_NAME,                           // name of your organization for display in the footer of messages
+  kmsEncryptedHookUrl: process.env.kmsEncryptedHookUrl, // encrypted slack webhook url
+  unencryptedHookUrl: process.env.unencryptedHookUrl,    // unencrypted slack webhook url
+  slackChannel: '#' + process.env.slackChannel,                 // slack channel to send a message to
+  slackUsername: process.env.slackUsername,               // "AWS SNS via Lamda", // slack username to user for messages
+  icon_emoji: process.env.icon_emoji,                      // slack emoji icon to use for messages
+  orgIcon: process.env.orgIcon,                           // url to icon for your organization for display in the footer of messages
+  orgName: process.env.orgName,                           // name of your organization for display in the footer of messages
+  awsAccountMap: process.env.awsAccountMap,              // JSON map of accountid:accountAlias
```

Comment:
Are these intentional or an artefact of some code formatter? Environment variables are typically uppercase.

Reply:
This was intentional to match the style and variable names of the existing slack notification function, which has env vars in lowerCamelCase. I'm happy to agree on ALL_CAPS for this as a style as I agree this is more typical.